tic_tac_toe/game.py

```diff
@@ -87,18 +162,61 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
-
+    if get_next_turn(game) == None:
+        raise InvalidMovement("Game is over.")
+    if get_next_turn(game) != player:
+        raise InvalidMovement("\"{}\" moves next".format(game['next_turn']))
```

Comment:
Could have just used single quotes to enclose the whole string then you wouldn't have to escape the double quotes inside the string.